### PR TITLE
fix Marcus' PR

### PR DESCRIPTION
**Pull request by hasanheroglu**

### CODE REVIEW

**Comment by hasanheroglu** [comment]
Only tested with no_sec Things but seemed okay.

File: package.json [hunk 1]
```diff
@@ -20,21 +20,25 @@
...
+        "eslint": "^8.35.0",
         "mocha": "^6.2.0",
         "prettier": "2.0.5",
-        "typescript": "3.9.7"
+        "typescript": "4.9.5"
     },
     "scripts": {
         "build": "tsc",
-        "start": "node dist/wot-test-bench.js",
+        "start": "node dist/src/wot-test-bench.js",
```
Comment (egekorkan) [inline package.json:37]: it should not generate it here. Could you fix tsconfig such that it is only under dist and not another level of src?

**Review by egekorkan — CHANGES_REQUESTED** [review]

**Comment by egekorkan** [comment]
Other than the small comment, it is looking good.

File: package.json (hunk 1 above)
Comment (hasanheroglu) [reply package.json]: Fixed at [f89dd79](https://github.com/tum-esi/testbench/pull/47/commits/f89dd7914846e9e0109569a6272fe941aea2f898)

**Pull request merged**